pkg/loki/loki.go

```diff
@@ -648,7 +651,7 @@ func (t *Loki) setupModuleManager() error {
 		Write:   {Ingester, Distributor},
 		Backend: {QueryScheduler, Ruler, Compactor, IndexGateway, BloomGateway, BloomCompactor},
 
-		All: {QueryScheduler, QueryFrontend, Querier, Ingester, Distributor, Ruler, Compactor},
+		All: {QueryScheduler, QueryFrontend, Querier, Ingester, Distributor, Ruler, Compactor, BloomCompactor},
```

Comment:
Up for discussion, but I don't think that we want to include the bloom compactor in the `all` target.
For testing, it comes handy, I understand.

Reply:
I don't think users of the monolith mode will benefit from building blooms significantly, right? In addition to that, it will increase resource usage and operational burden.

Reply:
Fair points all around, but we can leave it to enable better testing then remove it prior to release.